ldms/python/ldmsd/ldmsd_controller

```diff
@@ -292,18 +292,18 @@ class LdmsdCmdParser(cmd.Cmd):
         Load a plugin at the Aggregator/Producer
         Parameters:
         name=   The plugin to load
-        inst=     The configuration name for the plugin (default = <name>)
+        plugin= The configuration name for the plugin (default = <name>)
         """
         arg = self.handle_args('load', arg)
         if arg:
             name = arg['name']
-            if arg['inst']:
-                inst = arg['inst']
+            if arg['plugin']:
+                plugin = arg['plugin']
             else:
-                inst = name
-            rc, msg = self.comm.plugn_load(name, inst)
+                plugin = name
+            rc, msg = self.comm.plugn_load(name, plugin)
             if rc:
-                print(f'Error loading plugin {arg["name"]} as {arg["inst"]}: {msg}')
+                print(f'Error loading plugin {arg["name"]} as {arg["plugin"]}: {msg}')
```

Comment:
Might want to change this message a bit. "plugin X as Y" probably reads to a user as "X" being the real plugin name, and "Y" being the instance name.

I'd maybe suggest this:

```
   print(f'Error loading instance {arg["name"]} from plugin {arg["plugin"] if arg["plugin"] else arg[["name"]}: {msg}')
```